strax/run_selection.py

```diff
@@ -289,17 +303,24 @@ def define_run(self: strax.Context,
         raise ValueError(f"Can't define run from {type(data)}")
 
     # Find start and end time of the new run = earliest start time of other runs
-    run_md = dict(start=float('inf'), end=0, livetime=0)
+    run_md = dict(start=datetime.datetime.max.replace(tzinfo=pytz.utc), 
```

Comment:
could use a `defaultdict(int)` here

Reply:
I never worked with defaultdicts, but I think it does not really help here.  I would need different values for start and end because of,
```
run_md['start'] = min(run_md['start'], run_doc_start)
run_md['end'] = max(run_md['end'], run_doc_end)
```
would not I? Hence I do not see anything to gain.

Reply:
Right it would only be for the livetime where you'd actually be using it's behavior. There is nothing wrong with the current implementation 😉